pyalgotrading/algobulls/connection.py

```diff
@@ -271,20 +279,112 @@ def get_report(self, strategy_code, trading_type, report_type, render_as_datafra
             print('Report not available yet. Please retry in sometime')
             # return response
 
-    def backtest(self, strategy=None, start=None, end=None, instruments=None, lots=1, parameters=None, candle=None, mode=StrategyMode.INTRADAY, delete_previous_trades=True, **kwargs):
+    def get_pnl_report_table(self, strategy_code, trading_type):
         """
-        Submit a backtesting job for a strategy on the AlgoBulls Platform
+            Fetch BT/PT/RT Profit & Loss details
+
+            Args:
+                strategy_code: strategy code
+                trading_type: type of trades : Backtesting, Papertrading, Realtrading
+
+            Returns:
+                Report details
+        """
+        assert isinstance(strategy_code, str), f'Argument "strategy_code" should be a string'
+
+        # Fetch the data
+        data = self.get_report(strategy_code=strategy_code, trading_type=trading_type, report_type=TradingReportType.PNL_TABLE)
+
+        # Post-processing: Cleanup & converting data to dataframe
+        column_rename_map = OrderedDict([
```

Comment:
move this line inside the `if` cond on the next line.

Reply:
done